up_bridge/plexmo/dispatcher.py

```diff
@@ -76,3 +77,84 @@ def status(self) -> str:
     def set_replan_callback(self, callback):
         """Set callback function that triggers replanning"""
         self._replan_cb = callback
+
+
+class PlanDispatcher:
```

Comment:
`PlanDispatcher` and the old `SequentialPlanDispatcher` should be refactored as there is much overlap between them. Currently this is not represented in the code. 
I can do that after this PR is merged.

Reply:
Probably the `SequentialPlanDispatcher` is obsolete as it serves only a special case of parallel plan execution.